hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/BaseSparkCommitActionExecutor.java

```diff
@@ -292,8 +294,12 @@ protected void setCommitMetadata(HoodieWriteMetadata<HoodieData<WriteStatus>> re
   @Override
   protected void commit(HoodieWriteMetadata<HoodieData<WriteStatus>> result) {
     context.setJobStatus(this.getClass().getSimpleName(), "Commit write status collect: " + config.getTableName());
-    commit(result.getWriteStatuses(), result, result.getWriteStats().isPresent()
-        ? result.getWriteStats().get() : result.getWriteStatuses().map(WriteStatus::getStat).collectAsList());
+    if (!config.getBoolean(WRITE_IGNORE_FAILED) && result.getWriteStatuses().filter(status -> status.getErrors().size() > 0).count() > 0) {
+      throw new HoodieException("Error writing record in the job");
```

Comment:
Please add some logs to show error info.

Reply:
OK，had add first one of error info to exception details instead of add logs，because this would error out with details @beyond1920